packages/drawer/src/Drawer/Drawer.tsx

```diff
@@ -110,34 +112,48 @@ export const Drawer = forwardRef<HTMLDivElement, DrawerProps>(
             {...rest}
           >
             <div
-              className={getHeaderStyles({
-                hasShadowTop: !hasTabs && !isInterceptInView,
-                hasTabs,
+              className={getInnerContainerStyles({
+                displayMode,
                 theme,
               })}
             >
-              <Body
-                as={typeof title === 'string' ? 'h2' : 'div'}
-                baseFontSize={BaseFontSize.Body2}
-                id={titleId}
-                weight="medium"
+              <div
+                className={getHeaderStyles({
+                  hasShadowTop: !hasTabs && !isInterceptInView,
```

Comment:
nit: This check is repeated a few times: `!hasTabs && !isInterceptInView`. Should it be a `const`?